mec/mec1501/component/spi_periph.h

```diff
@@ -0,0 +1,341 @@
+/**
+ *
+ * Copyright (c) 2021 Microchip Technology Inc. and its subsidiaries.
+ *
+ * \asf_license_start
+ *
+ * \page License
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the Licence at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * \asf_license_stop
+ *
+ */
+
+/** @file spi_periph.h
+ *MEC1501 SPI Peripheral registers
+ */
+/** @defgroup MEC152x SPI peripheral device
+ */
+
+#ifndef _SPI_PERIPH_H
+#define _SPI_PERIPH_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include "regaccess.h"
+
+/* =========================================================================*/
```

Comment:
Can we drop these lines or use a simple comment?

Reply:
forgot but all other changes done.